runtime/syntax/man.vim

```diff
@@ -1,7 +1,8 @@
 " Maintainer:          Anmol Sethi <hi@nhooyr.io>
 " Previous Maintainer: SungHyun Nam <goweol@gmail.com>
-
-if exists('b:current_syntax')
+"
+if exists('b:current_syntax') || &filetype != 'man'
+  " May have been included by some other filetype.
   finish
```

Comment:
this defeats the purpose of any syntax file including man.vim syntax

Reply:
Can you explain? I simply moved the `&filetype != man` to before the highlighting is set.

Reply:
It's just what I said: there would be zero reason to include this file if you are skipping it here. The whole point of that check was to allow other syntax files to include this file and get the highlights, without the code _following_ the highlights.

Reply:
I have undone this change.

I originally included it because the Table of Contents buffer was sourcing the Man syntax highlighting, which I don't think is intended. Could be the issue lies elsewhere.